pkg/networkmanager/network-main.jsx

```diff
@@ -92,8 +93,8 @@ export const NetworkPage = ({ privileged, operationInProgress, usage_monitor, pl
 
         if (show_traffic) {
             const samples = usage_monitor.samples[iface.Name];
-            row.columns.push({ title: samples ? cockpit.format_bits_per_sec(samples[1][0] * 8) : "" });
-            row.columns.push({ title: samples ? cockpit.format_bits_per_sec(samples[0][0] * 8) : "" });
+            row.columns.push({ title: samples ? mock(cockpit.format_bits_per_sec(samples[1][0] * 8), "tx") : "" });
+            row.columns.push({ title: samples ? mock(cockpit.format_bits_per_sec(samples[0][0] * 8), "rx") : "" });
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20220819-083129-c0e512fa-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)